metabase/metabase
# Search Milestone 5: Add `created_at` search filter

`created_at` filter for Search Milestone 5: #27982 

### Description

Adds `created_at` filter to the search sidebar. The code uses the `DatePicker` component we already have to allow users to make date selections.


### How to verify

Go to the search sidebar, and try to filter by `Creation date`. You should only be able to see assets that were created in that date range. For example, create a Question, then filter the search by `Today` and you should see that question.

After filtering, try to refresh the page or paste the URL into a new window to test that hydrating the search filters still works when entering the search directly from a URL. The results should remain the same and should behave in the same way.

### Tests
